user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -46,9 +46,8 @@ class QualificationSummary:
     """
     total_apps: pd.DataFrame = field(init=True)  # Total apps, including failed or skipped
     tools_processed_apps: pd.DataFrame = None  # Apps after tools processing and heuristic filtering
-    recommended_apps: pd.DataFrame = None  # Apps recommended by legacy speedups. TODO: Should use QualX
-    filter_apps_count: int = field(default=0, init=False)  # Count after applying console filters (top candidates)
     top_candidates_flag: bool = False
```

Comment:
After this refactor, we probably do not need `top_candidates_flag: bool` in this class and it can be removed from the constructor. In the only usage below condition `self.filter_apps_count > 0` is sufficient.

https://github.com/NVIDIA/spark-rapids-tools/blob/f1d1e6cda8eec156e24cc975f7846b58d3e5ce4f/user_tools/src/spark_rapids_pytools/rapids/qualification.py#L134-L136